scripts/ubuntu-22.04-cpp.dockerfile

```diff
@@ -32,6 +29,8 @@ ADD scripts /velox/scripts/
 # are required to avoid tzdata installation
 # to prompt for region selection.
 ARG DEBIAN_FRONTEND="noninteractive"
+# Set a default timezone, can be overriden via ARG
+ARG tz="Europe/Madrid"
```

Comment:
I wonder why we set Europe/Madrid in first place ..

Reply:
Jacob added this file and he set Europe as the default.

Reply:
nit: I think its safe to remove this or maybe set it to UTC.

Reply:
We cannot remove the default arg. Ubuntu tzdata requires this as mentioned in the comment above.
I changed the default to `Etc/UTC`.
Reference https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image